samples/apps/smallbank/app/smallbank.cpp

```diff
@@ -41,6 +44,45 @@ namespace ccfapp
   private:
     SmallBankTables tables;
 
```

Comment:
Checking the content type on every message is nice for debugging/users, but arguably not necessary in a benchmark.

Reply:
Sure, I can leave the methods around for debugging purposes and remove the check from the test